### PR TITLE
Ran grammarly on introduction section

### PR DESCRIPTION
**Pull request by danich1**

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -3,36 +3,36 @@
...
-Scientific journals didn't become a predominant mode of communication until the 19th and 20th centuries when the first journal was created [@doi:10.1021/ci00050a001; @smith2006trouble; @doi:10.1300/J123v41n01_04].
+Scientific journals didn't become a predominant mode of communication until the 19th and 20th centuries, when the first journal was created [@doi:10.1021/ci00050a001; @smith2006trouble; @doi:10.1300/J123v41n01_04].
 Although scientific journals became the primary method of communication, they added high maintenance costs and long publication times to scientific discourse [@smith2006trouble; @doi:10.1300/J123v41n01_04].
-Some scientists’ solution to these issues has been to also communicate through preprints, which are scholarly works that have yet to undergo peer review process [@doi:10.1177/0192623318767322; @doi:10.1371/journal.pbio.2003995].
+Some scientists' solution to these issues has been to communicate through preprints, which are scholarly works that have yet to undergo peer review process [@doi:10.1177/0192623318767322; @doi:10.1371/journal.pbio.2003995].
```
Comment (cgreene) [inline content/02.introduction.md:8]: Should this be "solutions"?

File: content/02.introduction.md [hunk 2]
```diff
@@ -3,36 +3,36 @@
...
 
 Preprints are commonly hosted on online repositories, where users have open and easy access to these works.
 Notable repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different repositories available [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
 The burgeoning uptake of preprints in life sciences has been examined through research focused on metadata from the bioRxiv repository.
 For example, life science preprints are being posted at an increasing rate  [@doi:10.1371/journal.pbio.3000269].
 Furthermore, these preprints are being rapidly shared on social media, routinely downloaded, and cited [@doi:10.1371/journal.pone.0047523].
-Certain preprint categories are read and shared by both scientists and non-scientists alike [@doi:10.1101/2020.03.06.981589].
+Specific preprint categories are read and shared by both scientists and non-scientists alike [@doi:10.1101/2020.03.06.981589].
```
Comment (cgreene) [inline content/02.introduction.md:15]:
```suggestion
Some preprint categories are shared on social media by both scientists and non-scientists [@doi:10.1101/2020.03.06.981589].
```

File: content/02.introduction.md [hunk 3]
```diff
@@ -3,36 +3,36 @@
...
 About two-thirds to three-quarters of preprints are eventually published [@doi:10.7554/eLife.45133; @doi:10.1002/pra2.175] and life science articles that have a corresponding preprint version are cited and discussed more often than articles without them [@doi:10.1162/qss_a_00043; @doi:10.7554/eLife.52646; @doi:10.12688/f1000research.19619.2].
 Preprints take an average of 160 days to be published in the peer-reviewed literature [@doi:10.2139/ssrn.3455146], and those with multiple versions take longer to publish[@doi:10.2139/ssrn.3455146].
 
 The rapid uptake of preprints in the life sciences also poses challenges. 
 Preprint repositories receive a growing number of submissions [@doi:10.1371/journal.pgen.1008565].
-Linking preprints with their published counterparts is a key part of maintaining consistency of scholarly discourse but is challenging to perform manually [@doi:10.1038/530265a; @doi:10.7554/eLife.52646; @doi:10.1007/s11192-021-03900-7].
+Linking preprints with their published counterparts is vital in maintaining scholarly discourse consistency but is challenging to perform manually [@doi:10.1038/530265a; @doi:10.7554/eLife.52646; @doi:10.1007/s11192-021-03900-7].
```
Comment (cgreene) [inline content/02.introduction.md:21]:
```suggestion
Linking preprints with their published counterparts is vital to maintaining scholarly discourse consistency but is challenging to perform manually [@doi:10.1038/530265a; @doi:10.7554/eLife.52646; @doi:10.1007/s11192-021-03900-7].
```

File: content/02.introduction.md [hunk 4]
```diff
@@ -3,36 +3,36 @@
...
+Certain scientists have expressed concern that competitors could scoop them by making results available before publication [@doi:10.1371/journal.pgen.1008565; @doi:10.1371/journal.pbio.3000151].
 Preprint repositories by definition do not perform in-depth peer review, which can result in posted preprints containing inconsistent results or conclusions [@doi:10.12688/f1000research.19619.2; @doi:10.1007/s10393-018-1352-3; @doi:10.1038/530265a; @doi:10.1016/j.bpj.2016.06.035]; however, an analysis of preprints posted at the beginning of 2020 revealed that most underwent minor changes as they were published [@doi:10.1101/2021.02.20.432090].
-Despite a growing emphasis on using the study of preprints to examine the publishing process in the life sciences, how these findings relate to the text of all documents in bioRxiv has yet to be examined.
+Despite a growing emphasis on using preprints to examine the publishing process within life sciences, how these findings relate to the text of all documents in bioRxiv has yet to be explored.
```
Comment (cgreene) [inline content/02.introduction.md:26]:
```suggestion
Despite a growing emphasis on using preprints to examine the publishing process within life sciences, how these findings relate to the text of all documents in bioRxiv has yet to be examined.
```

**Review by cgreene — APPROVED** [review]
A couple small suggestions and one question

File: content/02.introduction.md (hunk 1 above)
Comment (danich1) [reply content/02.introduction.md:8]: I believe so, will change.

**Pull request merged**